pinecone_text/dense/openai_encoder.py

```diff
@@ -46,7 +48,9 @@ def __init__(
                 "dependencies by running: "
                 "`pip install pinecone-text[openai]"
             )
+        super().__init__(dimension=dimension)
         self._model_name = model_name
+        self._dimension = dimension
```

Comment:
this is redundant in case we add it to the base class

Reply:
Now I changed the logic so it is relevant